identity-iota/src/document/iota_document.rs

```diff
@@ -551,12 +543,10 @@ impl IotaDocument {
   /// - The signature verification operation fails.
   pub fn verify_document(signed: &IotaDocument, signer: &IotaDocument) -> Result<()> {
     // Ensure signing method is allowed to sign document updates.
-    signer
-      .verifier()
+    let options = VerifierOptions::default()
       .method_scope(MethodScope::capability_invocation())
-      .method_type(Self::UPDATE_METHOD_TYPES.to_vec())
-      .verify(signed)
-      .map_err(Into::into)
+      .method_type(Self::UPDATE_METHOD_TYPES.to_vec());
```

Comment:
Aside: it would be nice to have these options static. I tried making the functions in `VerifierOptions` const but ran into the lack of the `const_precise_live_drops` feature in stable to handle dropping the `MethodType` `Vec`. Other option is `lazy_static` but probably not worth it in this instance.

Reply:
Feel free to create an issue for this and we can look into it later, when/if `const_precise_live_drops` lands.